packages/validator/src/cli-validator/utils/get-copyright-string.js

```diff
@@ -0,0 +1,5 @@
+const getVersionString = require('./get-version-string');
+
+module.exports = function() {
+  return `IBM OpenAPI Validator (${getVersionString()}), @Copyright IBM Corporation 2017, 2023.\n`;
```

Comment:
I forgot to ask this in your previous PR but just out of curiosity - what was the motivation for including the copyright statement?

Reply:
Just from my experiences on various product teams, tools like that would typically display a copyright notice on the first line of output.  I think there might actually be some sort of guidelines around this topic that would tell us to log a copyright notice.
But I can easily take it out if it doesn't belong there.

Reply:
Oh I have no issue with it, I was just a little surprised. Sounds good to me